test/e2e/framework/pod/resize.go

```diff
@@ -335,10 +335,12 @@ func VerifyPodContainersCgroupValues(ctx context.Context, f *framework.Framework
 				cpuQuota = -1
 			}
 			expectedCPULimitString = strconv.FormatInt(cpuQuota, 10)
+			expectedCPULimitStringRounded := strconv.FormatInt(RoundUp(cpuQuota), 10)
 			expectedMemLimitString = strconv.FormatInt(expectedMemLimitInBytes, 10)
 			if *podOnCgroupv2Node {
 				if expectedCPULimitString == "-1" {
 					expectedCPULimitString = "max"
+					expectedCPULimitStringRounded = "max"
 				}
 				expectedCPULimitString = fmt.Sprintf("%s %s", expectedCPULimitString, CPUPeriod)
```

Comment:
Need to update the rounded string here too

Reply:
done